EmberMap animating across routes with Ember Animated (#299)
ember-learn/ember-blog: source/2019-10-04-the-ember-times-issue-118.md

---

## [EmberMap Video on Animating Across Routes with Ember Animated ✨](https://twitter.com/samselikoff/status/1175143786852364289)

> If you're interested in learning more about how to ✨**animate across routes** ✨ using the fantastic [Ember Animated addon](https://github.com/ember-animation/ember-animated) you may want to check out a new video by **EmberMap** in which [@samselikoff](https://github.com/samselikoff/) provides a great intro to the subject.

Reviewer: I'd add commas for clarity.
```suggestion
If you're interested in learning more about how to ✨**animate across routes** ✨ using the fantastic [Ember Animated addon](https://github.com/ember-animation/ember-animated), you may want to check out a new video by **EmberMap**, in which [@samselikoff](https://github.com/samselikoff/) provides a great intro to the subject.
```

> While this is a somewhat advanced topic, even if you have not previously worked with **Ember Animated**, the explanations are clear and easy to follow making it a decent starting place. So if you have been waiting to checkout Ember Animated this might be a good time to do so!

Reviewer: ```suggestion
While animation across routes is a somewhat advanced topic, even if you have not previously worked with **Ember Animated**, the explanations are clear and easy to follow, making the video a decent starting place. So if you have been waiting to check out Ember Animated, this might be a good time to do so!
```

> The video is available on both [Youtube](https://www.youtube.com/watch?v=O4Mt-dDqkk0) and the [EmberMap site](https://embermap.com/video/animating-across-routes-with-ember-animated). 

Reviewer: ```suggestion
The video is available on both [YouTube](https://www.youtube.com/watch?v=O4Mt-dDqkk0) and the [EmberMap site](https://embermap.com/video/animating-across-routes-with-ember-animated). 
```
